tailwind.config.ts

```diff
@@ -14,12 +14,28 @@ const config: Config = {
         'color-1': '#0f3b26', // Main Green
         'color-2': '#f9f7f7', // NavBar Off White
         'color-3': '#222', // Dark Gray
+        'txt-1': '#fff',
+        'txt-4': '#ddd',
         'active-1': '#4db6ac', // TODO: Decide on a color for this
         'hover-1': '#4db6ac', // TODO: Decide on a color for this
         'hover-2': '#89B4FA', // TODO: Decide on a color for this
         'event-banner': '#5E18EB', // Event Banner Background
         socials: '#7BD4CB', // Social
       },
+      spacing: {
+        '136': '30rem',
+      },
+      aspectRatio: {
+        event: '2 / 1', // This controls the aspect ratio of event elements, do not touch you will probably break stuff.
+      },
+      boxShadow: {
+        // Box Shadow - based on https://codepen.io/sdthornton/pen/wBZdXq
+        l1: '0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24)',
+        l1Hover: '0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22)',
```

Comment:
as with the first reasoning I don't think it would make things better here, I think when we are working with alpha its sometimes better to use `rgba`, the way I am looking at it is its best to work in the color pallette that displays what is changing and allows the easiest modification, hex works great for regular colors because we understand it and are working in hex, but these are just shades of black at different opacitys which makes rgba a lot more intuitive to use. (maybe @amberwelby could chime in on some thoughts here though).

Reply:
Woah didn't see this tag, mb. No strong thoughts here, I can see arguments for either. As long as we're consistent within this subsections I don't have a reason to object

Reply:
I think this is a case where its not about consistency but for reference when I sent a hex rgba the other day like `#00000040` it was confusing because not everyone is aware of the `a` part of the hex syntax.

Reply:
sounds good